universal-login-contracts/contracts/KeyHolder.sol

```diff
@@ -5,10 +5,12 @@ import "./IKeyHolder.sol";
 contract KeyHolder is IKeyHolder {
     mapping (address => Key) public keys;
 
```

Comment:
Make it public, so you don’ need getter later.

Reply:
Curiosity: why in ERC1077 _lastNonce is a private variable with a getter and not simply a public variable?

Reply:
Probably historical reasons. `public uint lastNonce;` would be good.